recipes/plasflow/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# install tensorflow version 0.10.0rc0
+pip install --no-deps --ignore-installed --no-cache-dir  tensorflow/tensorflow-0.10.0rc0-cp35-cp35m-linux_x86_64.whl
```

Comment:
Is there any reason you need this exact version? And if so is the runtime dependency on tensorflow still needed?

Reply:
yes, plasflow doesnt seem to work properly with the conda version, and its install instructions say to use this exact version (and I left tensorflow in the yaml file so that its dependencies would still be installed by conda and I could use `--no-deps`, but maybe better to just lists those explicitly then yeah?)